docs/04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enable Kyma with JetStream
+---
+
+This guide shows how to enable JetStream and how it ensures `at least once` delivery.
```

Comment:
```suggestion
This guide shows how to enable JetStream and how it ensures the at-least-once delivery.
```
Now that I'm thinking about it I guess we do not need the brackets at all.

Reply:
I think the brackets are good. But I don't think we need `the` before `at least once`

Reply:
No, we don't :) You're right :)